Make GnuPG signing optional

bastianeicher opened the pull request:
This can be useful for automated merging and testing of incoming in CI environments.

talex5 commented:
This looks useful, but I'm not sure about making `create` default to not using a key. That makes it very easy to create a non-working repository accidentally. Perhaps you should have to use `0repo create my-dir -` or something?

bastianeicher commented:
You're write. Running 0repo in this "fake" mode should be more explicit.
I've rebased the PR on the current master and changed to to use the `0repo create my-dir -` syntax you suggested.

talex5 commented:
Thanks!
